wordpress-mobile/gutenberg-mobile
# Correct android font weight

Fixes the issue with no effect on Android when `fontWeight` value is numerical.
More about the issue you can read [here](https://github.com/facebook/react-native/issues/26193)

Going through the comments and potential solutions I've created a function for mapping a font to font-weight called `correctTextFontWeight` which has to be called from the application starting point (`_src/index.js_`)

before | after
--- | ---
<img width="427" alt="Screenshot 2020-04-09 at 12 36 41" src="https://user-images.githubusercontent.com/22746080/78888421-dc05b780-7a61-11ea-95e7-101ff907613c.png"> | <img width="427" alt="Screenshot 2020-04-09 at 12 28 38" src="https://user-images.githubusercontent.com/22746080/78888460-f344a500-7a61-11ea-89e6-87e7320964a5.png">

before | after
--- | ---
<img width="427" alt="Screenshot 2020-04-09 at 13 01 29" src="https://user-images.githubusercontent.com/22746080/78888626-4dde0100-7a62-11ea-86ae-fb71e2b313b6.png"> | <img width="427" alt="Screenshot 2020-04-09 at 12 29 08" src="https://user-images.githubusercontent.com/22746080/78888647-59c9c300-7a62-11ea-886c-8e242a41e86a.png">

To test:

1. Open WP-Android app
2. Open PAGE draft
3. Expect correct font weight (thicker - 600) on template buttons texts

To test: 

1. Open playground app on Android
2. Scroll to unsupported block
3. Expect correct font weight (thicker - 600) on template buttons texts

To test: 

1. Open playground app on iOS
2. Expect app is not crashing

NOTE: I would like to add some unit test if you like that approach.

PR submission checklist:
